src/data/nat/choose/central.lean

```diff
@@ -97,4 +98,37 @@ lemma four_pow_le_two_mul_self_mul_central_binom : ∀ (n : ℕ) (n_pos : 0 < n)
 calc 4 ^ n ≤ n * central_binom n : (four_pow_lt_mul_central_binom _ le_add_self).le
 ... ≤ 2 * n * central_binom n    : by { rw [mul_assoc], refine le_mul_of_pos_left zero_lt_two }
 
+lemma two_dvd_central_binom_succ (n : ℕ) : even (central_binom (n + 1)) :=
```

Comment:
People are usually extremely literal in lemma names, so please either rephrase this in the form `2 ∣ central_binom (n + 1)` or rename to `even_central_binom_succ` (either is fine).

Reply:
I rephrased to the former as that allowed me to remove the import.